src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import React from 'react';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: React.Dispatch<React.SetStateAction<Tab>>,
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  let selectedTab = tabs[0];
+
+  function handleTabSelect(isTabSelected: boolean, tab: Tab) {
+    if (!isTabSelected) {
+      onTabSelected(tab);
+    }
+  }
+
+  return (
+    <div>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isTabSelected = selectedTabId === tab.id;
+
+            if (isTabSelected) {
+              selectedTab = tab;
+            }
+
+            return (
+              <li
+                className="is-active"
```

Comment:
add some styles for this class, I don't see the selected tab
![image](https://user-images.githubusercontent.com/37161315/180442911-1591f9c8-cb8b-4550-ba1c-98448d3b95ee.png)